target/linux/bmips/base-files/etc/board.d/02_network

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+. /lib/functions/uci-defaults.sh
+
+board_config_update
+
+case "$(board_name)" in
+comtrend,ar-5387un)
+	ucidef_add_switch "switch0" \
```

Comment:
Since this is a new target, I suggest use upstream B53 DSA driver instead of swconfig.

Reply:
@dengqf6 unfortunately, dsa b53 and bcm63xx-enetsw drivers needs some modifications to get it working, and I would need help from @KanjiMonster / @ffainelli for that...

Reply:
Marking this as resolved since I've removed downstream ethernet drivers.